src/pages/workspaces/workspace/WorkspaceMenu.test.js

```diff
@@ -250,8 +285,6 @@ describe('WorkspaceMenu - Azure workspace', () => {
   })
 
   it('renders menu item Leave as enabled', () => {
-    // Arrange
-    useWorkspaceDetails.mockReturnValue({ workspace: { workspace: {} } })
```

Comment:
This was a bug-- the workspace details being provided mimic a GCP workspace instead of an Azure one. The default `forEach` setup at [line 245](https://github.com/DataBiosphere/terra-ui/pull/3449/files#diff-851ce7a2d69a65c25533c9610d2285c83b610774441e147a53580f460dd2e57cR245) is sufficient here.